telemetry/data_window.go

```diff
@@ -117,6 +362,12 @@ func getWindowData() []*windowData {
 	i := 0
 	for i < len(rotatedSubWindows) {
 		thisWindow := *rotatedSubWindows[i]
+		var startWindow windowData
+		if i == 0 {
+			startWindow = thisWindow
+		} else {
+			startWindow = *rotatedSubWindows[i-1]
+		}
```

Comment:
So it is iterating like `thisWindow = r[1] - r[0]`, `thisWindow = r[2] - r[0]`, `thisWindow = r[3] - r[1]` for `thisWindow.SQLUsage.SQLTotal = rotatedSubWindows[i].SQLUsage.SQLTotal - startWindow.SQLUsage.SQLTotal`?

I don't quite understand what you want to do, but is the distance of `2` between windows intended?